package.json

```diff
@@ -88,7 +88,8 @@
     "moment": "^2.29.2",
     "os-locale": "^6.0.2",
     "stream-json": "^1.7.4",
-    "tar-fs": "^2.1.1"
+    "tar-fs": "^2.1.1",
+    "got": "^12.1.0"
```

Comment:
any reason to not use the latest one ? [12.3.1](https://www.npmjs.com/package/got/v/12.3.1)

Reply:
updated